### DISCUSSION TITLE
All displayed values are wrong

### DISCUSSION THREAD

**Discussion by alexanderbjelos** [tbnobody/OpenDTU#865] [Q&A]
For months OpenDTU reliably shows me all values from my Hoymiles 600. Since a few days all values are wrong. I can not determine until today whether it is my inverter or OpenDTU. I have already restarted everything several times. Do any of you have any ideas what else I could do? Otherwise I would create a bug.
![IMG_9562](https://user-images.githubusercontent.com/22572905/235359260-97b5f398-348d-4bb4-9f80-e2d1ff27ae98.jpeg)

**Comment by Solarteur** [comment]
- Which firmware version is installed?
- Can you see the LED on the inverter? (flashing green?)
- Does the inverter produce electricity?
- Have you tested a different location for the DTU?
- Can you post the console output here?

**Comment by DPO99** [comment]
Dieser Fehler tritt doch immer dann auf wenn man eine Wechselrichter bei 2 Dtu (opendtu, ahoy, original HM) gleichzeitig einbindet.

**Reply by ghost** [reply]
Das kann nicht der einzige Grund sein. Bei mir gibt es definitiv nur eine einzige (open)DTU, trotzdem habe ich auch schon absurd falsche Werte erlebt. Zum Beispiel wurden für einen DC-String mal 230,4 V und 204,84 A angezeigt. Ich habe den Eindruck, dass solche "Halluzinationen" vor allem nach Ende der Produktionsphase (d.h. rotes Inverter-Feld) anzutreffen sind.
(Möglicherweise liegt es an einer aufgrund von nachlassender DC-Leistung schwächelnden oder unvollendeten Funkübertragung, deren verstümmelte Daten dann trotzdem als vermeintlich regulär übernommen wurden? Der Effekt könnte dem von zwei sich gegenseitig störenden DTUs ähneln.)

**Comment by alexanderbjelos** [comment]
@Solarteur & @DPO99 First of all, thank you very much for your support. I spent a few more hours of troubleshooting yesterday. The error was indeed the antenna module. After I replaced the component, everything works just fine.
